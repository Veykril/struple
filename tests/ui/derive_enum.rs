use struple::Struple;

#[derive(Struple)]
enum Foo {
    Bar(i32, u64, String),
    Baz,
}

fn main() {}
