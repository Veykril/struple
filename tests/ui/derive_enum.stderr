error: Struple may only be derived on structs
 --> tests/ui/derive_enum.rs:3:10
  |
3 | #[derive(Struple)]
  |          ^^^^^^^
  |
  = note: this error originates in the derive macro `Struple` (in Nightly builds, run with -Z macro-backtrace for more info)
